examples/truffle/truffle-config.js

```diff
@@ -1,12 +1,17 @@
-//! Simple truffle configuration with configured develop network for testing
-//! our rust examples.
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+
+const {
+  PK,
+  INFURA_PROJECT_ID,
+  ETHERSCAN_API_KEY
+} = process.env;
 
 module.exports = {
   networks: {
-    develop: {
```

Comment:
With this gone, how will you be able to develop locally now?

Reply:
Yes, see #46